Profiles/Sassy/responses.yml

```diff
@@ -842,6 +846,9 @@ Chat:
   ModeratorClosedGuessingGameBeforeStarting:
   - Text: Pay attention, {0}. We haven't even started yet
   - Text: Close what, {0}? The guessing game never started.
+  - Text: I know, {0}. I'm ready for this to be over too. But we haven't started the guessing game yet.
+  - Text: I think {0} is telling you to go faster as they wanted to start the guessing game
+  - Text: Are you even watching, {0}? We have started the guessing game yet.
```

Comment:
I assume you meant "We have<ins>n't</ins> started"?

Reply:
Maybe one day I'll think to proofread my stuff before creating PRs!